rest/api.go

```diff
@@ -225,7 +225,9 @@ func (r *muxAPI) StartTLS(addr Address, certFile, keyFile FilePath, middleware .
 func (r *muxAPI) preprocess() {
 	r.validateRulesOrPanic()
 	if r.config.GenerateDocs {
-		newDocGenerator().generateDocs(r)
+		if err := newDocGenerator().generateDocs(r); err != nil {
+			log.Fatal(err)
+		}
```

Comment:
This is the primary change. I noticed that the `_docs` generation would sometimes silently fail and found the culprit was this line.

If you prefer, I could make this `log.Println()` instead? @tylertreat-wf